proposals/demonstration-organisation.md

```diff
@@ -0,0 +1,163 @@
+   - `https://github.com/oscal-compass-demos/Australian-govt-ISM` # provides sufficient context.
+   - `https://github.com/oscal-compass-demos/kubecon-japan-2025` # detailed for an event. 
+4. Secondary repositories begin with the name of the demonstration
+   - e.g `https://github.com/oscal-compass-demos/acme-corp-cloud-catalog`
+
+5. Demonstration content MUST be stable. 
+   - E.g. Owners MUST NOT update the repos under `https://github.com/oscal-compass-demos` in order to run the demo, the demonstration SHOULD be forked.
+6. Demonstration repositories can be nominated via an issue in the  [`community`](https://github.com/oscal-compass/community)
+   1. Demonstrations MUST have at two maintainers nominated.
+   2. The name of the demonstration; a description of the intent of the demonstrations; and the repositories are required
+7. Demonstrations MUST be actively maintained to the currently supported versions of `oscal-compass`
+8. Demonstrations that are not maintained will either be:
+   1. Archived, or
+   2. Actively document to user why there are limits (e.g. relies on 3rd party content that leverages an old version of OSCAL).
+9.  Demonstrations must include appropriate blerbs and cross linking from:
+   1.  Oscal Compass community website
+   2.  and optionally project websites (e.g. `compliance-trestle` or `compliance-to-policy`)
+10. Event specific demonstrations (e.g. Kubecon) should follow the following practice
+    1.  The SHOULD be kept stable with respect to the documentation and videos
+    2.  If reused in another event they MUST be renamed with another project name
+    3.  If reused and renamed the events it is used at MUST be referenced in the README.md for traceability
+
+
+## Design Details
+
+Sufficient details are above.
+
+## Impacts / Key Questions
+
+The proposal is relatively simple.
+
+The biggest question is how much needs to be recorded in the document itself as opposed to being extracted from the history on GitHub (such as approvers).
+
+
+### Pros
+- Provides clear guidance to curate demonstrations
+- Keeps the core organisation clear
+- Provides clear expectations for both demo developers and consumers.
+- Provides expectations to demo developers and maintainers of required ongoing efforts.
+
+### Cons
+
+- It's additional overhead from an OSCAL compass organization perspective which carries the risk of being ignored.
+
+
+## Risks and Mitigations
+
+N/A
+
+### Security Considerations
+
+- Github Actions automation carries risks based on how authorization is performed. This proposal will decrease risk as it isolates demo automation further away from core oscal compass projects.
+
+- The ongoing risk with authoritative demonstrations is that it is a larger body of code and projects 
+## Future Milestones (Optional)
+
+None
+
+## Implementation Details (Optional) 
+
+1. Setup the organization via the CNCF with the correct set of permissions.
```

Comment:
It might good to look at existing projects for norms around this use case to define whether the demos should be part of the core OSCAL Compass project or a separate downstream of the OSCAL Compass project. We would need to go through certain steps of the on-boarding process again to add another organization to the CNCF provided infrastructure.